dtfs-central-api/src/v1/controllers/portal/gef-deal/update-deal.js

```diff
@@ -0,0 +1,44 @@
+const { generateAuditDatabaseRecordFromAuditDetails } = require('@ukef/dtfs2-common/change-stream');
```

Comment:
I've just moved this file from the `update-deal.controller.js` since it is now used outside of the controller.

Reply:
Could this be moved out of the `controllers` folder?

Reply:
Good shout @MarRobSoftwire how about a new helpers directory? So we'd have:

```
central-api/src/v1/controllers/*
central-api/src/v1/helpers/portal/update-gef-deal
```

Reply:
I think ideally it would be in the repositories folder, but we dont want to go down the rabbit hole of making it consistent with the other repo classes. I'd be happy wherever you want to move it

Reply:
No need for this anymore now that we have a portal activity repo. Have reverted the change. Resolving my comment